surveys/urls.py

```diff
@@ -10,4 +10,9 @@
         views_api.SurveyPostView.as_view(),
         name='survey-post',
     ),
+    url(
+        r'^forms/(?P<pk>\d+)/respond/(?P<user_id>\w+)?$',
```

Comment:
It could be completely arbitrary, we should support as many characters as possible I think (do I need to do `.*` or whatever the correct syntax is?)